modules/openapi-generator-core/src/main/java/org/openapitools/codegen/api/AbstractTemplatingEngineAdapter.java

```diff
@@ -49,8 +49,8 @@ protected String[] getModifiedFileLocation(String location) {
      */
     private String getPathWithoutExtension(String location) {
         if (location == null) return null;
-        int idx = location.lastIndexOf('.');
-        if (idx == -1) return location;
-        return location.substring(0, idx);
+        int indexOfPath = location.lastIndexOf('.');
```

Comment:
Shouldn't this be called `indexOfExtension` instead?

Reply:
yes making the required changes.

Reply:
@martin-mfg pushed the required changes.